multinet/uploaders/swagger/nested_json.yaml

```diff
@@ -0,0 +1,29 @@
+Upload a nested JSON file to a collection of tables
+---
+consumes:
+  - text/plain
+
+parameters:
+  - $ref: "#/parameters/workspace"
+  - $ref: "#/parameters/table"
+  - name: data
+    in: body
+    description: Raw JSON content of file
+    schema:
+      type: object
+
+responses:
+  200:
+    description: JSON uploaded to tables
+    schema:
+      type: object
+      properties:
+        edgecount:
+          type: integer
+        int_nodecount:
+          type: integer
+        leaf_nodecount:
+          type: integer
+
```

Comment:
This doesn't return 400 but maybe it should. It looks like the nested json doesn't do any validation. Should we open an issue here?

Reply:
Yes, let's open an issue. It's probably a low-priority thing unless we run into problems with processing nested_json files, which we probably won't since the process for getting data into evo-bio has changed (and no longer uses nested_json files).

Reply:
Looks like #159 already exists :)